test-requirements.txt

```diff
@@ -4,3 +4,5 @@ oslotest
 ipdb
 nose==1.3.7
 nose-testconfig==0.9.1
+sqlalchemy
```

Comment:
The jobs fails because pip found double requirements in `requirements.txt` and `test-requirements.txt`
Remove it or wait #908 is merged